src/spec-common/dotfiles.ts

```diff
@@ -44,20 +44,34 @@ export async function installDotfiles(params: ResolverParameters, properties: Co
 ${createFileCommand(markerFile)} || (echo dotfiles marker found && exit 1) || exit 0
 command -v git >/dev/null 2>&1 || (echo git not found && exit 1) || exit 0
 [ -e ${targetPath} ] || ${allEnv}git clone ${repository} ${targetPath} || exit $?
-if [ -x ${installCommand} ]
+echo Setting current directory to ${targetPath}
+cd ${targetPath}
+
+if [ -f ${targetPath}/${installCommand} ]
```

Comment:
With `cd ${targetPath}` above you can cover the absolute and relative path both with `if [ -f "${installCommand}" ]` and use the `else` to exit with an error message because the file should exist.

Reply:
Agreed, I thought that would be covered by the `cd` you added. 👍

Reply:
I've tried several techniques but I think the way I have it now is the most correct, otherwise it will be unhappy with one of the following examples:

1. `install.sh`
2. `./install.sh`
3. `/home/node/dotfiles/install.sh`